doc/internal/virtualenv.rst

```diff
@@ -104,7 +104,7 @@ Setting up
 
 Make sure you download and install:
 
-    - `Python 3.6+  <http://www.python.org/downloads/windows/>`_
+    - `Python 3.7+  <http://www.python.org/downloads/windows/>`_
```

Comment:
Maybe we can omitt the version here, so we don't have to keep updating it in the future.